.github/CODEOWNERS

```diff
@@ -7,6 +7,7 @@
 
 # Community partners
 /algorithmia/                            @koverholt support@algorithmia.io
+/superwise/                              @oryan-omer support@superwise.ai
```

Comment:
Can we move this so the file remains in alphabetical order?

Reply:
Done